openhands/core/config/sandbox_config.py

```diff
@@ -32,6 +32,10 @@ class SandboxConfig:
         browsergym_eval_env: The BrowserGym environment to use for evaluation.
             Default is None for general purpose browsing. Check evaluation/miniwob and evaluation/webarena for examples.
         platform: The platform on which the image should be built. Default is None.
+        port_mappings: Custom port mappings from container ports to host ports. Default is empty dict.
+            This is a dictionary where keys are container ports and values are host ports.
+            For example, {8080: 8080} maps container port 8080 to host port 8080.
+            Note: Port 4141 is always mapped to host port 4141 by default.
```

Comment:
Similar to VSCode, maybe we need to add something like `vscode_url` to runtime class for a list of accessible URL / port? Otherwise this FE will only work for eventstream runtime.